src/DropdownMenu.tsx

```diff
@@ -204,17 +204,19 @@ const DropdownMenu: DropdownMenu = React.forwardRef(
       (menuProps as any).alignRight = alignEnd;
     }
 
+    let style = (props as any).style;
```

Comment:
Using `let` is usually for variables, but this is a constant. I would recommend you to use `const`.

Reply:
This is the correct usage. I am reassigning the var in a block below.